trainee_template/train.amlg

```diff
@@ -408,6 +421,9 @@
 									))
 									(accum (assoc trained_instance_count 1 ))
 
+									;auto analyze if appropriate
+									(call !AutoAnalyzeIfNeeded)
```

Comment:
should be inside an `(if autoAnalyzeEnabled`

Reply:
ahh good eye